database/migration/src/main/resources/changesets/embiggen_detritus_value.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="rmunshi" id="embiggen_call_caching_detritus_entry">
+        <modifyDataType tableName="CALL_CACHING_DETRITUS_ENTRY"
+                        columnName="DETRITUS_VALUE"
+                        newDataType="LONGTEXT"/>
```

Comment:
To avoid possible future relational mapping problems, the various classes under the `cromwell.database` mappings should be kept in sync to convert this field from a VARCHAR to a CLOB. Otherwise we might "have a bad time" with slick or another ORM/FRM truncating your column data.

Also TOL: When I have time, I need to go back and figure out why we used LONGVARCHAR for hsqldb CLOBs in the past and not LONGTEXT...